skglm/utils/prox_funcs.py

```diff
@@ -204,3 +204,77 @@ def prox_SLOPE(z, alphas):
             x[i] = d
 
     return x
+
+
+@njit
+def log_1p_exp_vec(x, eps):
+    """Map log(1 + exp(x) / eps) to vector."""
+    return np.log(1 + np.exp(x) / eps)
```

Comment:
IIRC this can lead to instabilities, maybe https://numpy.org/doc/stable/reference/generated/numpy.log1p.html fixes them

more detailed resource: https://fa.bianp.net/blog/2019/evaluate_logistic/

Reply:
I changed it. Thanks for sharing the resources!